src/stacked_borrows.rs

```diff
@@ -684,9 +684,9 @@ impl Stacks {
         state: &GlobalState,
     ) -> InterpResult<'tcx> {
         trace!("deallocation with tag {:?}: {:?}, size {}", tag, alloc_id, range.size.bytes());
-        let mut state = state.borrow_mut();
+        let state = state.borrow_mut();
```

Comment:
Does this mean we can just `borrow` here?

Reply:
ah that works, too, I'll fix it :smile: